Zend/zend_interfaces.stub.php

```diff
@@ -58,7 +58,7 @@ public function unserialize(string $data);
 interface Countable
 {
     /** @return int */
-    public function count();
+    public function count(); // TODO Can't add tentatie return type due to DOMNodeList::count() and DOMNamedNodeMap::count()
```

Comment:
I've eliminated the false return for these methods and added the tentative types, so this should be fine now.